Code/Tools/ProjectManager/Source/GemCatalog/GemRequirementFilterProxyModel.cpp

```diff
@@ -24,7 +24,8 @@ namespace O3DE::ProjectManager
     {
         // Do not use sourceParent->child because an invalid parent does not produce valid children (which our index function does)
         QModelIndex sourceIndex = sourceModel()->index(sourceRow, 0, sourceParent);
-        return GemModel::IsAdded(sourceIndex) && GemModel::HasRequirement(sourceIndex);
+        const bool hasNewVersion = !GemModel::GetNewVersion(sourceIndex).isEmpty();
+        return GemModel::IsAdded(sourceIndex) && GemModel::HasRequirement(sourceIndex) && (!GemModel::WasPreviouslyAdded(sourceIndex) || hasNewVersion) ;
```

Comment:
nitpick extra space before semicolon
```suggestion
        return GemModel::IsAdded(sourceIndex) && GemModel::HasRequirement(sourceIndex) && (!GemModel::WasPreviouslyAdded(sourceIndex) || hasNewVersion);
```